jsk_robot_common/jsk_robot_startup/scripts/shutdown.py

```diff
@@ -24,22 +25,33 @@ class Shutdown(object):
 
     def __init__(self):
         rospy.loginfo('Start shutdown node.')
+        self.client_jp = SoundClient(sound_action='/robotsound_jp', blocking=True)
         rospy.Subscriber('shutdown', Empty, self.shutdown)
         rospy.Subscriber('reboot', Empty, self.reboot)
         self.shutdown_command = rospy.get_param(
             '~shutdown_command', '/sbin/shutdown -h now')
         self.reboot_command = rospy.get_param(
             '~reboot_command', '/sbin/shutdown -r now')
+        self.volume = rospy.get_param('~volume', 1.0)
+
+    def speak(self, client, speech_text, lang=None):
```

Comment:
What about setting the default value of lang to `''`?
It will eliminate subsequent IF branches.
`client.say(speech_text, voice=lang, volume=self.volume, replace=False)`

Reply:
That's a good idea! I fixed it.

Reply:
Thanks!